### ISSUE TITLE
pub get/upgrade are broken in 2.0.0-dev.18.0

### ISSUE THREAD

**Issue by kevmoo** [dart-lang/sdk#31940]
This is the SDK tracking sister to https://github.com/dart-lang/pub/issues/1781

Should be closed by the change that lands the DEPS update containing the fix

```
package pub_dartlang_org requires SDK version >=1.25.0-dev.9.0 <2.0.0 but the current SDK is 2.0.0-dev.18.0.
```

Getting a lot of red on Travis

https://travis-ci.org/dart-lang/dart_style/builds/330425079
https://travis-ci.org/dart-lang/pub-dartlang-dart/builds/330406465

**Comment by Hixie** [comment]
When this is fixed, please let Flutter know so that we can unpin the Dart version we're using on Travis (engine side).

**Comment by matanlurey** [comment]
Is it possible to get an update here?

This is blocking the ...
* The flutter engine roll (https://github.com/flutter/engine/pull/4558)
* AngularDart release (https://github.com/dart-lang/angular/issues/789)

... and virtually every travis run for every package is now red (that has not pinned).

**Comment by munificent** [comment]
https://dart-review.googlesource.com/c/sdk/+/35803

**Comment by whesse** [comment]
Fix is released in version 2.0.0-dev.19.0
